apps/nsqd/nsqd.go

```diff
@@ -119,6 +119,7 @@ func nsqdFlagSet(opts *nsqd.Options) *flag.FlagSet {
 	flagSet.Duration("max-output-buffer-timeout", opts.MaxOutputBufferTimeout, "maximum client configurable duration of time between flushing to a client")
 	flagSet.Duration("min-output-buffer-timeout", opts.MinOutputBufferTimeout, "minimum client configurable duration of time between flushing to a client")
 	flagSet.Duration("output-buffer-timeout", opts.OutputBufferTimeout, "default duration of time between flushing data to clients")
+	flagSet.Int("max-channel-client-connection-count", opts.MaxChannelClientConnectionCount, "maximum channel client connection count")
```

Comment:
Let's name this `max-channel-subscribers`?

Reply:
Actually that name isn't right either, because it's only on a per-node basis.

Thinking out loud... how about `max-local-channel-subscriptions`?

Reply:
agreed, could use a better name here

Reply:
I think `max-channel-subscriptions` should be fine as other command line arguments are also in this pattern but they all apply to single nsqd instance.

Reply:
@mreiferson @ploxiln What do you think. Naming is the top one difficult problem. :)

Reply:
My take: `max-channel-consumers` and explain the "local" aspect in the description. But I'll defer to @mreiferson :)

Reply:
In nsq world, consumer is more official than subscription. I will prefer `max-channel-consumers`

Reply:
SGTM